roda-app.rb

```diff
@@ -69,7 +72,10 @@ class RodaApp < Roda
       r.on "reservations" do
         r.route 'reservation'
       end
-
+      # /api/orders
+      r.on "orders" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.